app/Models/Task.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Task extends Model
+{
+    use HasFactory;
+    protected $fillable = [
```

Comment:
use guarded instead of fillable

```suggestion
    protected $guarded = ['id'];
```